src/lib/edge-functions/proxy.mjs

```diff
@@ -115,6 +115,7 @@ export const initializeProxy = async ({
 
     // Setting header with geolocation and site info.
     req.headers[headers.Geo] = Buffer.from(JSON.stringify(geoLocation)).toString('base64')
+    req.headers[headers.DeployID] = '0'
```

Comment:
This is the value we use for the `DEPLOY_ID` environment variable in Netlify Dev.